### PR TITLE
DWR-902 Added example queries and 'purging the data' section

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: docs/Runbook.BulkDeleteExams.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ As defined in [Import and Migrate](Runbook.migrate.md#create-update), the data u
 
 #### Before executing the scripts
 1. Stop Exam Processors.
-2. Execute [validation queries](https://github.com/SmarterApp/RDW_Schema/validation-scripts) to reconcile warehouse and reporting data mart(s).
+2. Execute [validation script](https://github.com/SmarterApp/RDW_Schema/blob/develop/validation/README.md) to reconcile warehouse and reporting data mart(s).
```
Comment (agorina) [inline docs/Runbook.BulkDeleteExams.md]: Is this path correct? (The `develop` concerns me)

File: docs/Runbook.BulkDeleteExams.md [hunk 2]
```diff
@@ -88,12 +101,66 @@ Purging exams includes:
...
+4. Delete exams:
+```sql
+DELETE e FROM exam e WHERE e.deleted = 1;
+```
+
+### Post-validating exam deletion
+5. Count the number of records and compare it to the count from the step 2 above. The numbers must match.
+```sql 
+SELECT count(*) FROM exam;
+```
```
Comment (agorina) [inline docs/Runbook.BulkDeleteExams.md]: @malaffoon , is this enough of the information?

File: docs/Runbook.BulkDeleteExams.md [hunk 3]
```diff
@@ -88,12 +101,66 @@ Purging exams includes:
...
+### Purging from reporting data mart
 If exams were deleted using soft-delete and migration (see above) there is no need to do this: the system has already deleted the records in the data mart.
+
+To delete exams bypassing the soft-delete step, follow instructions for [Purging from data warehouse](#purging-from-warehouse). Skip step 3.3 since `reporting` data mart does not have this table.
+
+### Purging from OLAP data mart
 (TODO)
 
 ### Purging exam auditing data in the warehouse
 (TODO)
```
Comment (agorina) [inline docs/Runbook.BulkDeleteExams.md]: I am planning on leaving the above TODOs in here and creating a new JIRA to address them when we are closer to the release date. @jtreuting and @malaffoon, Do you have any objections?

File: docs/Runbook.BulkDeleteExams.md (hunk 1 above)
Comment (malaffoon) [reply docs/Runbook.BulkDeleteExams.md]: Yeah, i've been struggling with that. I try really hard to have only relative links but can't do that across repos. For now, i think this is the only thing you can do.

File: docs/Runbook.BulkDeleteExams.md (hunk 3 above)
Comment (malaffoon) [reply docs/Runbook.BulkDeleteExams.md]: nope

File: docs/Runbook.BulkDeleteExams.md (hunk 2 above)
Comment (malaffoon) [reply docs/Runbook.BulkDeleteExams.md]: sure

**Pull request merged**